contracts/TypedMemView.sol

```diff
@@ -225,8 +277,16 @@ library TypedMemView {
         }
     }
 
-    /// Unsafe raw pointer construction. This should generally not be called
-    /// directly. Prefer `ref` wherever possible.
+    /**
+     * @notice          Unsafe raw pointer construction. This should generally not be called
+     *                  directly. Prefer `ref` wherever possible.
+     * @dev             Unsafe raw pointer construction. This should generally not be called
+     *                  directly. Prefer `ref` wherever possible.
+     * @param _type     The type
+     * @param _loc      The memory address
+     * @param _len      The length
+     * @return          newView - The new view with the specified type, location and length
+     **/
     function buildUnchecked(uint256 _type, uint256 _loc, uint256 _len) private pure returns (bytes29 newView) {
```

Comment:
Would propose adding `unsafe` to the title of any function that is unsafe.

Reply:
Not sure how much of a headache that would be.  Would have to bump version and add documentation of changes in README for sure

Reply:
given that this is private, we could rename without any effect. Go ahead and make that change